### ISSUE TITLE
Githubs viability long term

### ISSUE THREAD

**Issue by sgrieve** [LSDtopotools/LSDTopoTools2_roadmap#7]
As the rumors of GH getting bought by Microsoft are intensifying, it might be worth using this as an opportunity to talk about long term archiving of lsd development.

The Turing institute use https://github.com/josegonzalez/python-github-backup to run nightly archiving of all of the content of their repos.

I'm not saying we abandon GH, but it might make sense to think about what we do if things start going south in the future.

**Comment by dvalters** [comment]
Lots of other options around too: gitlab, bitbucket etc. An archive-style backup/mirror might be good.

Tangent: Seems like a lot of hysteria on Slack/Twitter etc...  I mean, GitHub Inc was a private venture-capital funded company before...was never a non-profit etc :stuck_out_tongue_closed_eyes: Even if they did start charging, would it be that unreasonable to *pay* for a software service and cloud hosting for code... [devil's advocate :smiling_imp:]

**Comment by sgrieve** [comment]
Yeah, I agree that there is a lot of overreaction at the moment. I just feel that from a software archiving point of view a platform that could undergo radical change with little to no notice  isn't the safest place to store what is becoming the work of people's whole careers.

**Comment by dvalters** [comment]
yes good point. I have one repo at the minute with two origins - one on github and one on bitbucket. When you push to origin it gets mirrored to the mirror origin. So that is one (accidental...) solution.

**Comment by dvalters** [comment]
(Though by virtue of git being a distributed VCS, you have some sort of (last resort) backup already from people's local clones which have the whole repo history.)
